src/main/java/org/example/homework5/Task1.java

```diff
@@ -0,0 +1,25 @@
+package org.example.homework5;
+
+public class Task1 {
+    public static void main(String[] args) {
+        int input = 280;
+        int evenDigitSum = getEvenDigitSum(input);
+        System.out.println("For input number " + input + " sum of even digits is " + evenDigitSum);
+    }
+
+    private static int getEvenDigitSum(int number) {
+        if (number < 0) {
+            return -1;
+        }
+        int result = 0;
+        int lessDigit = number;
```

Comment:
I don't really understand what is `lessDigit`?)
Looks like you can just keep on using `number`, isn't it?

Reply:
This variable was there to keep input parameter unchanged. As it was uselesss in thismethod I've refactoredcode to remove it